lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -1545,7 +1545,7 @@ struct LowerVectorExtractStridedSliceOpAIEv1Pattern
   LogicalResult
   matchAndRewrite(vector::ExtractStridedSliceOp extractOp, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
-    auto vType = extractOp.getVectorType();
+    auto vType = extractOp.getVector().getType();
```

Comment:
The new type access method is `getSourceVectorType`. In theory, in your change, `auto` resolves to `Type` instead of `VectorType`, but it looks like `getRank` on `vType` is not failing. I guess there's some static polymorphism shenanigans at play but, for safety, let's stick with the original semantics.